content/issues/2021-04-14-maintenance-publick8s.md

```diff
@@ -0,0 +1,19 @@
+---
+title: AKS publick8s version update
+date: 2021-04-14T07:30:00-00:00
+resolved: true
+resolvedWhen: 2021-04-14T10:00:00-00:00
+# Possible severity levels: down, disrupted, notice
+severity: disrupted
```

Comment:
```suggestion
severity: notice
```

Reply:
IMHO, it should be disrupted because ldap.jenkins.io, infra.ci and release.ci will be restarted in the process